include/seqan3/range/view/interleave.hpp

```diff
@@ -0,0 +1,382 @@
+ * | std::ranges::CommonRange        |                                       | *preserved*                     |
+ * | std::ranges::OutputRange        |                                       | *preserved*                     |
+ * | seqan3::const_iterable_concept  |                                       | *preserved*                     |
+ * |                                 |                                       |                                 |
+ * | seqan3::reference_t             |                                       | seqan3::reference_t<urng_t>     |
+ *
+ *
+ * If above requirements are not met, this adaptor forwards to
+ * `| ranges::view::chunk(step_size) | ranges::view::join(inserted_range)`
+ * which returns a view with the following properties:
+ *
+ * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)  |
+ * |---------------------------------|:-------------------------------------:|:-------------------------------:|
+ * | std::ranges::InputRange         | *required*                            | *preserved*                     |
+ * | std::ranges::ForwardRange       | *required*                            | *lost*                          |
+ * | std::ranges::BidirectionalRange |                                       | *lost*                          |
+ * | std::ranges::RandomAccessRange  |                                       | *lost*                          |
+ * | std::ranges::ContiguousRange    |                                       | *lost*                          |
+ * |                                 |                                       |                                 |
+ * | std::ranges::ViewableRange      | *required*                            | *guaranteed*                    |
+ * | std::ranges::View               |                                       | *guaranteed*                    |
+ * | std::ranges::SizedRange         |                                       | *lost*                          |
+ * | std::ranges::CommonRange        |                                       | *lost*                          |
+ * | std::ranges::OutputRange        |                                       | *lost*                          |
+ * | seqan3::const_iterable_concept  |                                       | *lost*                          |
+ * |                                 |                                       |                                 |
+ * | seqan3::reference_t             |                                       | seqan3::value_type_t<urng_t>    |
+ *
+ * * `urng_t` is the type of the range modified by this view (input).
+ * * `rrng_type` is the type of the range returned by this view.
+ * * for more details, see \ref view.
+ *
+ * ### Example
+ *
+ * \snippet test/snippet/range/view/interleave.cpp general
+ * \hideinitializer
+ */
+inline auto constexpr interleave = detail::interleave_fn{};
```

Comment:
the convention here changed again so it`s `inline constexpr auto interleave = ...`

Reply:
Just wondering, does the order actually make a difference? Or is this more for consistency's sake.

Reply:
No, it's just a matter of style.